integration/setup/setup.go

```diff
@@ -134,6 +138,19 @@ func setupCollection(tb testing.TB, ctx context.Context, client *mongo.Client, o
 			continue
 		}
 
+		// if validators are set, create collection with them (otherwise collection will be created on first insert)
+		if validators := provider.Validators(*handlerF); validators != nil {
+			opts := new(options.CreateCollectionOptions)
+			for key, value := range validators {
+				if *handlerF == "tigris" {
```

Comment:
We pass handlerF to Validators method, but then do extra changes there. Why? May we pass both handler and collection name to Validators?

Reply:
Fixed